src/lib/CookieDecoder.ts

```diff
@@ -90,7 +94,7 @@ export class CookieDecoder {
             label: walletLabel,
             accounts,
             contracts: [],
-            deleted: false,
+            keyMissing: false,
```

Comment:
You forgot to remove the default `false` here. But that this passes the test, means the test is broken. So first fix that the test reports an error here.

Reply:
The test works now. It tests 6 wallets now instead of 3 to cover every combination of KeyType and keyMissing Status.